app/views/idv/shared/_doc_capture_interstitial.html.erb

```diff
@@ -8,18 +8,30 @@
       ) %>
 <% end %>
 
+<div class="text-center margin-bottom-4">
+  <%= image_tag asset_url('idv/interstitial_icons.svg'), width: 207, height: 88, alt: '', class: 'display-block margin-x-auto' %>
+</div>
+
 <%= render PageHeadingComponent.new do %>
-  <%= t('doc_auth.headings.verify_with_phone') %>
+  <%= t('doc_auth.headings.document_capture') %>
 <% end %>
 
 <% if decorated_sp_session.sp_name.present? %>
   <p>
-    <%= t('doc_auth.info.login_access_sp', app_name: APP_NAME, sp_name: decorated_sp_session.sp_name) %>
+    <%= t('doc_auth.info.socure_hybrid_flow', app_name: APP_NAME, sp_name: decorated_sp_session.sp_name) %>
   </p>
-<% end %>
+<% else %>
+  <p>
+    <%= t('doc_auth.info.socure_hybrid_flow', app_name: APP_NAME, sp_name: APP_NAME) %>
+  </p>
+  <% end %>
+
+<p>
+  <%= t('doc_auth.info.socure_basic_idv', app_name: APP_NAME) %>
+</p>
 <p>
-  <%= t('doc_auth.info.add_id_consent_with_phone', app_name: APP_NAME) %>
+  <%= t('doc_auth.info.socure_ial2_idv_finish', app_name: APP_NAME) %>
 </p>
 
-<%= link_to t('forms.buttons.continue'), @url, class: 'usa-button usa-button--big usa-button--wide' %>
-<%= render 'idv/doc_auth/cancel', step: 'verify_id' %>
+<%= link_to t('forms.buttons.continue'), @url, class: 'usa-button usa-button--big usa-button--wide margin-top-4' %>
```

Comment:
Could you check and adjust bottom margin to match? Should be 2.5rem/40px total top and bottom

Reply:
Screenshot looks good, all set now thank you! 

(In Figma, we have a little 8px padding on the Continue button + 32px between elements to get to 40px... but I think the implemented margins and padding is different. Doesn't matter to me as long as the space is even overall!)